File: modul-3/PRAK205-2210817210012-MuhammadRakaAzwar.py
import math
a, b = input().split()
a = float (a)
b = float (b)
print(f"Alas = {math.sqrt(b*b-a*a)} cm")
print(f"Tinggi = {a} cm", )
print(f"Keliling = {a+b+math.sqrt(b*b-a*a)} cm")
print(f"Luas = {0.5*math.sqrt(b*b-a*a)*a} cm^2")

File: modul-3/PRAK205-2210817210012-MuhammadRakaAzwar.c
#include <stdio.h>
int main() {
float a, b;
printf(" ");
scanf("%f%f", &a, &b);
printf("Alas = %.f cm\n", sqrt(b*b-a*a));
printf("Tinggi = %.f cm\n", a);
printf("Keliling = %.f cm\n", a+b+sqrt(b*b-a*a));
printf("Luas = %.f cm^2\n", 0.5*sqrt(b*b-a*a)*a);
}
